src/main/webapp/css/style.css

```diff
@@ -648,3 +648,8 @@ table.no-borders-warning caption {
 .jch-entries-per-page .jenkins-button, .jch-pagination .jenkins-button {
     min-width: 52px;
 }
+
+.jch pre {
+  background-color: var(--white);
```

Comment:
I thought of using `--background` but there is a lot more work to be done to make this work with the Dark theme.